### DISCUSSION TITLE
On the suitability of using the Bogoliubov transformed Hamiltonian to compute finite-temperature spectral functions

### DISCUSSION THREAD

**Discussion by jiangtong1000** [shuaigroup/Renormalizer#157] [Announcements]
I think the Bogoliubov transformation assumes a non-entangled product state and does not account for the entanglement that are essential for describing the thermal equilibrium. While the finite-temperature spectral functions should be evaluated w.r.t the thermal equilibrium density matrix. So is it suitable to use it here? 
https://github.com/shuaigroup/Renormalizer/blob/958332431b157ce0d2a704c6561a106ee9f62003/renormalizer/transport/spectral_function.py#L39-L43

**Comment by jjren** [comment]
If the initial state is entangled between the electronic and vibrational parts (e.g. the emission spectrum of molecular aggregates), I don't think  Bogoliubov transformation can work. In this case, the imaginary time evolution algorithm should be used. When the initial state is disentangled, it may work and probability be more efficient than what you have done based on the ITE algorithm.
